charts/grafana/FLUX.md

```diff
@@ -1,12 +1,29 @@
 # Flux language setup
-### Manual Setup
-* Flux is still in beta, so this information may be out of date
-* Follow instructions laid out by [Flux Datasource for Grafana]
-(https://grafana.com/plugins/grafana-influxdb-flux-datasource)
+## Manual Setup
+- [Flux](https://docs.influxdata.com/flux/v0.12/) is still in beta, so this information may be out of date
+- [Enable](https://docs.influxdata.com/flux/v0.12/introduction/installation) Flux by setting the `flux-enabled` option to `true` under the `http` section of the influxdb.conf
+- Follow instructions laid out by [Flux Datasource for Grafana](https://grafana.com/plugins/grafana-influxdb-flux-datasource)
 
+## Cronus-Monitoring Setup
+- Run the init.sh script in the scripts directory and the included helm chart will enable flux automatically
+    - Flux is enabled in Grafana via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/grafana/values.yaml) `datasources` with `grafana-influxdb-flux-datasource` as the `type` 
```

Comment:
again, what?

Reply:
"`datasources` section"???

Reply:
"Flux is enabled in Grafana via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/grafana/values.yaml), in the `datasources` section with" ...

Reply:
I changed it with the style you provided, take a look and see if thats good enough?